src/libraries/System.Globalization.Calendars/System.Globalization.Calendars.sln

```diff
@@ -1,4 +1,8 @@
-﻿Microsoft Visual Studio Solution File, Format Version 12.00
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.3.32922.545
```

Comment:
Please remove sln edits from this change...I believe we generate these.

Reply:
This has been corrected